client/components/NetworkList.vue

```diff
@@ -82,10 +135,61 @@ export default {
 		Channel,
 		Draggable,
 	},
+	data() {
+		return {
+			searchText: "",
+			activeSearchItem: null,
+		};
+	},
 	computed: {
+		activeChannel() {
+			return this.$store.state.activeChannel;
+		},
 		networks() {
 			return this.$store.state.networks;
 		},
+		items() {
+			// Array of [network, channel] tuples
+			const items = [];
+
+			for (const network of this.$store.state.networks) {
+				for (const channel of network.channels) {
+					if (
+						this.$store.state.activeChannel &&
+						channel === this.$store.state.activeChannel.channel
+					) {
+						continue;
+					}
+
+					items.push([network, channel]);
+				}
+			}
+
+			// Sort by most unreads
+			items.sort((a, b) => {
+				return b[1].unread - a[1].unread;
```

Comment:
Stabilize the sort by name when unreads equal

Reply:
As discussed in IRC I removed sorting so that we have the same predictable order as in the normal sidebar list.